Tools/PRWatcher.ps1

```diff
@@ -0,0 +1,193 @@
+#Streamlines Winget-pkgs manifest PR moderator approval by watching the clipboard - copy a PR title to your clipboard, and Watch-PRTitles attempts to parse the PackageIdentifier and version number, gathers the version from Winget, and gives feedback in your Powershell console. Also outputs valid titles to a logging file. Freeing moderators to focus on approving and helping. 
+
+
+function Watch-PRTitles {
+	[CmdletBinding()]
+	param(
+		[switch]$noNew,
+		$LogFile = ".\PR.txt",
+		$authFile = ".\Auth.csv",
+		$oldclip = ""
+	)
+	while($true){
+		$timevar = (get-date -Format T) + ":"
+		$clip = Get-Clipboard;
+		$AuthList = gc $authFile | ConvertFrom-Csv
+		$carryClip = ""
+		$copyClip = $false
+		
+		if (diff $clip $oldclip) {
+			$noRecord = $false
+			$title = $clip -split ": "
+
+			if ($title[1]) {
+				$title = $title[1] -split " "
+			}else {
+				$title = $title -split " "
+			}
+			#Split the title by spaces. Try extracting the version location as the next item after the word "version", and if that fails, use the 2nd to the last item, then 3rd to last, and 4th to last. For some reason almost everyone puts the version number as the last item, and GitHub appends the PR number. 
```

Comment:
The reason is most people use YamlCreate.ps1 or wingetcreate to author a manifest and they automatically generate PR titles of the form:

`New version|New package|Add|Update|Remove: <PackageIdentifier> version <PackageVersion>`